package.json

```diff
@@ -66,7 +66,7 @@
     "@bpmn-io/properties-panel": "^3.13.0",
     "@carbon/react": "^1.42.1",
     "@carbon/styles": "^1.42.1",
-    "@playwright/test": "1.39.0",
+    "@playwright/test": "^1.40.0",
```

Comment:
Was there a reason we pinned that? I can't really remember.

Reply:
I just have a habit of pinning deps :sweat_smile: